zds/utils/models.py

```diff
@@ -36,11 +36,11 @@ class Meta:
         verbose_name = 'Categorie'
         verbose_name_plural = 'Categories'
 
-    title = models.CharField('Titre', max_length=80)
+    title = models.CharField('Titre', unique=True, max_length=80)
```

Comment:
je disais ça à cause du unique, qui implique un index de choix.

Le 21/03/2016 20:50, Laville Augustin a écrit :
>
> In zds/utils/models.py 
> <https://github.com/zestedesavoir/zds-site/pull/3261#discussion_r56884917>:
>
> > @@ -36,11 +36,11 @@ class Meta:
> >          verbose_name = 'Categorie'
> >          verbose_name_plural = 'Categories'
> >
> > -    title = models.CharField('Titre', max_length=80)
> > +    title = models.CharField('Titre', unique=True, max_length=80)
>
> En soit non parce que j'importe automatiquement les nouvelles catégories.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/zestedesavoir/zds-site/pull/3261/files/8433acf22778c61de97283eb30df2b190543bf4d#r56884917>
>